src/scenes/PlantsDashboardRouter/components/LiveDeadPlantsPerSpeciesCard.tsx

```diff
@@ -88,8 +80,12 @@ export default function LiveDeadPlantsPerSpeciesCard({
             label=''
             options={allSpecies}
             selectedValue={selectedSpecies}
-            className={classes.maxDropdownWidth}
             fullWidth={true}
+            selectStyles={{
+              inputContainer: {
+                maxWidth: '228px',
```

Comment:
is this working correctly? Because inputContainer style is added to the div that contains the input og the dropdown, but the className is added to the div that wraps the whole component (outside the label)

Reply:
Thank you flagging this 🙌 I was hoping it would and I believe I checked the component looked right in Storybook, but  I'll take another look and make sure this is working and fix if not.

Reply:
Just realized I actually checked a similar change in another component, but not this component. Will still take another look and fix if needed.

Reply:
I think you may want to add another "select style" prop like in [this PR](https://github.com/terraware/web-components/commit/415e6ba5b9900296881c214f1ad50bd6fce9fe9a#diff-845080ba173a86b66b8d07537880ffd2a1e0136b6f2a6505f5a1a843be016ae8R11-R17) for `selectContainer` and then apply it to [this](https://github.com/terraware/web-components/commit/415e6ba5b9900296881c214f1ad50bd6fce9fe9a#diff-845080ba173a86b66b8d07537880ffd2a1e0136b6f2a6505f5a1a843be016ae8R235) top level div.

Reply:
Ok I just tested this change locally and it appears to limit the width of the dropdown 🎉 Thanks again for flagging Constanza, I like confirming changes work as expected 🙌